From acc6b81fcb421b32fb78faf3d55d2e834b7f66a3 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Tue, 16 Sep 2014 20:19:15 -0400
Subject: [fs] nfsd4: convert comma to semicolon

Message-id: <1410898849-22126-72-git-send-email-bfields@redhat.com>
Patchwork-id: 95817
O-Subject: [Patch RHEL7 071/165] nfsd4: convert comma to semicolon
Bugzilla: 1044798 1123721 1141932
RH-Acked-by: Benjamin Coddington <bcodding@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Himangi Saraogi <himangi774@gmail.com>

Replace a comma between expression statements by a semicolon. This changes
the semantics of the code, but given the current indentation appears to be
what is intended.

A simplified version of the Coccinelle semantic patch that performs this
transformation is as follows:
// <smpl>
@r@
expression e1,e2;
@@

 e1
-,
+;
 e2;
// </smpl>

Signed-off-by: Himangi Saraogi <himangi774@gmail.com>
Acked-by: Julia Lawall <julia.lawall@lip6.fr>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: fc8e5a644c2041273a1cee7c6299713ccee319ab
Bugzilla: 1044798
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfs4callback.c b/fs/nfsd/nfs4callback.c
index 564d723..c393d6c 100644
--- a/fs/nfsd/nfs4callback.c
+++ b/fs/nfsd/nfs4callback.c
@@ -678,7 +678,7 @@ static int setup_callback_client(struct nfs4_client *clp, struct nfs4_cb_conn *c
     (clp->cl_cred.cr_flavor >= RPC_AUTH_GSS_KRB5))
    return -EINVAL;
   args.client_name = clp->cl_cred.cr_principal;
-  args.prognumber = conn->cb_prog,
+  args.prognumber = conn->cb_prog;
   args.protocol = XPRT_TRANSPORT_TCP;
   args.authflavor = clp->cl_cred.cr_flavor;
   clp->cl_cb_ident = conn->cb_ident;
-- 
1.7.1